### ISSUE TITLE
how to change .ts extension on output m3u8

### ISSUE THREAD

**Issue by siberkolosis** [warren-bank/HLS-Proxy#38]
want to bypass firewall. how to change or rename .ts extension on m3u8 playlist output?

thank you.

**Comment by warren-bank** [comment]
this is a feature that wasn't previously available.

so I just released [v3.4.2](https://github.com/warren-bank/HLS-Proxy/commit/29f3db5d768d43109651a89f28797fe0fc9a62d6) which adds options to configure the file extensions used for both manifests (default: m3u8) and media segments (default: ts).

for example:

1. start server
   ```bash
      hlsd -v 3 --port 8080 --manifest-extension "manifest" --segment-extension "segment"
   ```
2. open master manifest directly in browser: [CBS News](https://www.cbsnews.com/common/video/cbsn_header_prod.m3u8)
3. open the same master manifest in browser through hls proxy: [modified_master.m3u8](http://127.0.0.1:8080/aHR0cHM6Ly93d3cuY2JzbmV3cy5jb20vY29tbW9uL3ZpZGVvL2Nic25faGVhZGVyX3Byb2QubTN1OA==.manifest)
4. open the 640p child manifest in browser through hls proxy: [modified_child.m3u8](http://127.0.0.1:8080/aHR0cHM6Ly9jYnNuLXVzLmNic25zdHJlYW0uY2JzbmV3cy5jb20vb3V0L3YxLzU1YTg2NDhlOGYxMzRlODJhNDcwZjgzZDU2MmRlZWNhL21hc3Rlcl8xMS5tM3U4.manifest)
5. open the first video segment in browser through hls proxy: [segment_1.ts](http://127.0.0.1:8080/aHR0cHM6Ly9jYnNuLXVzLmNic25zdHJlYW0uY2JzbmV3cy5jb20vb3V0L3YxLzU1YTg2NDhlOGYxMzRlODJhNDcwZjgzZDU2MmRlZWNhL21hc3Rlcl8xMV8yMzQyODQwMC50cz9tPTE2NTcwMzc2ODk=.segment)

**Comment by siberkolosis** [comment]
thank you very much... its working now..
